iati/core/resources/test_data/202/valid_dateorder.xml

```diff
@@ -4,5 +4,9 @@
   <planned-disbursement type="1">
```

Comment:
The rule name has an underscore in its name. Consistency with the name of test files would aid readability slightly.